persia/ctx.py

```diff
@@ -415,7 +415,6 @@ def __init__(
         sparse_optimizer: Optimizer,
         dense_optimizer: torch.optim.Optimizer,
         device_id: int = 0,
-        mixed_precision: bool = False,
```

Comment:
I think this should be called fp16_embedding?
Mixed precision is already named for https://pytorch.org/docs/stable/amp.html which we also want to support later

Reply:
The default setting is `fp16_embedding`.  `fp16_embedding` argument will be add on when implement `f32_embeding` later on.